packages/react-core/package.json

```diff
@@ -55,7 +62,7 @@
   "devDependencies": {
     "@patternfly/patternfly": "4.219.2",
     "@rollup/plugin-commonjs": "^21.0.0",
-    "@rollup/plugin-node-resolve": "^13.0.0",
+    "@rollup/plugin-node-resolve": "^15.0.1",
```

Comment:
Needed the latest @rollup/plugin-node-resolve as older versions did not recognize package.json `exports`.